readme.md

```diff
@@ -30,6 +30,11 @@ http: {
 * Indentation based syntax
```

Comment:
I didn't change this, but typically whitespace indentation languages do not use delimiters. I would remove this as a design goal even if it is *desirable* that within composite types the primitive values are indented for readability.

Reply:
Hmm.. Let's discuss more this on a call (will schedule one soon)

In general, given that we want to support annotations, indentation along delimiters (`{}`) makes the format more readable

Though the below example will give an impression that the one without delimiters look clean and simple, I think when you want to support arrays and multiline strings - a consistent delimiter structure approach of multimap (`{}`),  arrays (`[]`) and multiline strings (`()`) would be better

Would love to discuss further in a call.

#### Without delimiters
```groovy
http:
  method: 'GET'
  url: 'https://www.usebruno.com/hello'
  headers:
    Content-Type: 'application/json'

    @disabled
    @description('This is a sample request')
    Authorization: 'Bearer{{token}}'
  param:
    query:
      @description('The status of the user')
      @enum('active', 'inactive')
      status: 'active'
```

#### With delimiters 
```groovy
http: {
  method: 'GET'
  url: 'https://www.usebruno.com/hello'
  headers: {
    Content-Type: 'application/json'

    @disabled
    @description('This is a sample request')
    Authorization: 'Bearer{{token}}'
  }
  param: {
    query: {
      @description('The status of the user')
      @enum('active', 'inactive')
      status: 'active'
    }
  }
```

PS: The above example does not take into consideration of how an indentation based approach without delimiters would work in case of arrays and multi-line strings

Reply:
I prefer *with* delimiters (it also makes parsing much easier); I am recommending that indentation not be identified as syntactically significant (except where noted for multi-line strings) but preferred for readability.